src/features/hotspots/setup/HotspotSetupExternalConfirmScreen.tsx

```diff
@@ -61,12 +62,34 @@ const HotspotSetupExternalConfirmScreen = () => {
   }, [params])
 
   const navNext = useCallback(async () => {
-    navigation.push('HotspotSetupLocationInfoScreen', {
-      addGatewayTxn: params.addGatewayTxn,
-      hotspotAddress: publicKey,
-      hotspotType: params.hotspotType,
+    const solAddress = Account.heliumAddressToSolAddress(address || '')
```

Comment:
Is address always a helium address? If you send a solana address into heliumAddressToSolAddress will it return the same thing?

Reply:
yeah, because it comes from the wallet link token which is always helium